crates/image_viewer/src/image_viewer.rs

```diff
@@ -144,8 +144,13 @@ impl Item for ImageView {
             .map(Icon::from_path)
     }
 
-    fn breadcrumb_location(&self, _: &App) -> ToolbarItemLocation {
-        ToolbarItemLocation::PrimaryLeft
+    fn breadcrumb_location(&self, cx: &App) -> ToolbarItemLocation {
+        let display_breadcrumb = EditorSettings::get_global(cx).toolbar.breadcrumbs;
+        if display_breadcrumb && self.show_toolbar() {
```

Comment:
I do not think we need to check `show_toolbar` here, this should be enough:

```rust
let show_breadcrumb = EditorSettings::get_global(cx).toolbar.breadcrumbs;
if show_breadcrumb {
    ToolbarItemLocation::PrimaryLeft
} else {
    ToolbarItemLocation::Hidden
}
```

Reply:
ayy! alright, lemme update that

Reply:
looks good now @bennetbo?